src/nginxconfig/i18n/de/templates/app.js

```diff
@@ -36,4 +36,5 @@ export default {
     globalConfig: 'Globale Konfiguration',
     setup: 'Setup',
     configFiles: 'Konfigurationsdateien',
+    copied: 'Gekopieerd',
```

Comment:
This doesn't seem to be German?

Reply:
Sorry, I'll change it with the corresponding traduction.